stock_quant_package_dimension/models/stock_quant_package.py

```diff
@@ -180,11 +180,12 @@ def _compute_estimated_pack_weight_kg(self):
             picking_id
         )
         for package in self:
-            weight = 0.0
+            # Note: also include packaging weight
+            weight = package.product_packaging_id.weight
```

Comment:
```suggestion
            weight = package.product_packaging_id.base_weight
```
It is `base_weight` on v16 https://github.com/odoo/odoo/blob/16.0/addons/stock/models/stock_package_type.py#L22

Reply:
It is `package.package_type_id.base_weight` or `package.product_packaging_id.package_type_id.base_weight` (NOTE: `package_type_id` is not required)

Reply:
For a package that you delivery, the type of package is `package.package_type_id` and it's weight is given by `base_weight`

What is `package.product_packaging_id` ?

Reply:
Hello everyone,

After a thorough review of the issue, I am confident that the solution lies in utilizing `package.package_type_id.base_weight`. This is because the weight that needs to be added to the total weight corresponds to the weight of the specific package itself. For instance, if a pallet weighs 5 kgs, you must include this 5 kg in the total weight calculation.